scripts/subsetter.py

```diff
@@ -18,9 +18,11 @@
 parser.add_argument('--outdir', type=str,
     default=None,
     help='output directory')
+parser.add_argument('--compress_level', type=int,
```

Comment:
```suggestion
parser.add_argument('--compress-level', type=int,
```
Personally, I prefer this style, but up to you.